data/sideload.gschema.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schemalist>
+  <schema path="/io/elementary/sideload/" id="io.elementary.sideload" gettext-domain="io.elementary.sideload">
+    <key name="trash-flatpakref" type="b">
```

Comment:
I wonder if we should make this less specific in case we want to do the same with Flatpak bundles, repo files, etc.

```suggestion
    <key name="trash-on-success" type="b">
```